### DISCUSSION TITLE
Album Art not pulled from server

### DISCUSSION THREAD

**Discussion by zipzagster** [music-assistant/support#3062] [Q&A]
I can no longer get album art, as the URL point to Tidal, not the server. My tablets don't have internet access. Before this summer I could pull from the server. Here's where on album art is pointed:

https://resources.tidal.com/images/febc7760/486a/4634/87ac/a0a379725847/750x750.jpg

**Comment by OzGav** [comment]
Why is it necessary that you posted three times about the same thing?

**Comment by zipzagster** [comment]
This issue is unresolved. I travel. I work on the issue when I'm home and have time.I don't sit on the forum.

**Comment by zipzagster** [comment]
Stop closing my unresolved issue. I work on the problem when I’m not traveling.
